samples/sampler/stories/default/Slider.js

```diff
@@ -72,6 +72,7 @@ export const _Slider = () => {
 			)}
 			showAnchor={boolean('showAnchor', SliderConfig)}
 			step={number('step', SliderConfig, 1)}
+			wheelInterval={number('wheelInterval', SliderConfig, 200)}
```

Comment:
```suggestion
			wheelInterval={number('wheelInterval', SliderConfig)}
```

To avoid any confusion, let's just use default value.

Reply:
It's fixed.